Misc/NEWS.d/next/Library/2018-07-25-00-40-14.bpo-34213.O15MgP.rst

```diff
@@ -0,0 +1,2 @@
+Frozen dataclasses didn't work properly with an attribute called "object".
+Now they do.
```

Comment:
Please change to: Allow frozen dataclasses to have a field named "object". Previously this conflicted with an internal use of "object".